cli/src/deploy/mod.rs

```diff
@@ -1,13 +1,13 @@
+use dialoguer::{Confirm, Input, Select};
 use hyper::{Request, StatusCode};
+use indicatif::ProgressBar;
 use serde::{Deserialize, Serialize};
-use serde_json::{from_reader, from_str, json, Value};
-use spinner::SpinnerBuilder;
+use serde_json::{from_reader, from_str, json, to_writer_pretty, Value};
```

Comment:
can keep the calls for `from_reader`, `from_str`, and `to_writer_pretty` with the full path? With such ambiguous names it's a bit better for readability imo

We can also `use serde_json as json` so it would be `json::from_reader`, etc

Reply:
hmm I can do a full import for the three you mentioned, but I won't do `use serde_json as json` because it will interfere with `json` which is called much more often in this file